README.md

```diff
@@ -1 +1,62 @@
-# Capstone2-kanban
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Dynasty Entertainment.
+
+> A list of movies are shown and you can add a comment on them or like them.
+
+## [A brief about the project](https://drive.google.com/file/d/1hcB7XQrD2Pwnj7o61Nxnc9TL4wojA7fu/view?usp=sharing)
+
+
+## Built With
+
+- HTML, CSS & JavaScript.
+- webpack, bootstrap, fontawesome
+
+
+## live preview
+Not Available yet.
+
+
+## Getting Started
+
+### Prerequisites
+
+> Any Web Browser
+
+
+### Setup
+- clone the directory.
+- `npm i`.
+- `npm start`.
+
+
+## Authors
+
+👤 **Mo'athal S. Kachi**
+👤 **Hassan Momanyi**
+
+- GitHub: [@githubhandle](https://github.com/Moathal)
+- GitHub: [@githubhandle](https://github.com/fullstop125)
+
```

Comment:
- [ ] `required`
- [ ] Kindly document this properly `why?` because someone can't tell how many authors are here and also add the social handles.
- [ ] see the [template](https://github.com/microverseinc/readme-template)